server/handlers/image_handler.py

```diff
@@ -1,19 +1,53 @@
+import logging
+import socket
 from random import random, randint, choice
 import string
 
 
 class ImageHandler:
-    def __init__(self, gs):
+    def __init__(self, gs, config):
+        self.logger = logging.getLogger("main")
         self.gs = gs
+        self.config = config
+        print("╠ CREATED IMAGE HANDLER")
+        self.logger.info("╠ CREATED IMAGE HANDLER")
 
+    def socket_connect(self):
```

Comment:
I realized that in general, you may want to have sockets be `socketio` sockets. https://python-socketio.readthedocs.io/en/latest/intro.html#what-is-socket-io. They're probably better than bare sockets.